stream.go

```diff
@@ -161,6 +173,16 @@ func (cs *clientStream) SendMsg(m interface{}) (err error) {
 
 func (cs *clientStream) RecvMsg(m interface{}) (err error) {
 	err = recv(cs.p, cs.codec, m)
+	defer func() {
+		// err != nil indicates the termination of the stream.
+		if EnableTracing && err != nil {
+			cs.traceInfo.tr.Finish()
```

Comment:
this needs to be *after* the two calls below. cs.traceInfo.tr cannot be used after Finish is called.